public/app/features/query/components/QueryEditorRowHeader.tsx

```diff
@@ -85,7 +86,7 @@ export const QueryEditorRowHeader = <TQuery extends DataQuery>(props: Props<TQue
   return (
     <>
       <div className={styles.wrapper}>
```

Comment:
nit: we can skip rendering the entire wrapper if `hideName === true`

Reply:
This wrapper is enclosing elements that should be rendered independently of the `hideName` value.

Reply:
Ah I must have missed the other bits that are being rendered in here 👍